Objects/Objects/BuiltElements/Beam.cs

```diff
@@ -136,6 +136,12 @@ public class ArchicadBeam : Beam
     public string? elementType { get; set; } /*APINullabe*/
     public List<Classification>? classifications { get; set; } /*APINullabe*/
 
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public Base? elementProperties { get; set; }
+
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public Base? componentProperties { get; set; }
```

Comment:
Do we respect `NullValueHandling` in our serialiser?

If I understand this correctly, this causes the property to not be added to the `json` when null, but that may be counter productive for data analysis scenarios like Excel and PowerBI where we don't have an object model available.

Is this necessary for some reason?

Reply:
The reason is not to export empty null values. OK, since this info is relevant for PowerBI, I removed that NullValueHandling.

Reply:
Perfect thx!